hana/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.0.7](https://github.com/cap-js/cds-dbs/compare/hana-v0.0.6...hana-v0.0.7) (2024-03-12)
+
```

Comment:
we need to add https://github.com/cap-js/cds-dbs/pull/488 → release please seems to be buggy if a conventional commit contains code fences as described here: https://github.com/googleapis/release-please/issues/2020